Translations update from Kodi Weblate

weblate opened the pull request:
Translations update from [Kodi Weblate](https://kodi.weblate.cloud) for [Kodi web interfaces/TeX](https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/).



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widgets/kodi-web-interfaces/-/tex/horizontal-auto.svg)

ahtuyi00-dotcom commented:
> Translations update from [Kodi Weblate](https://kodi.weblate.cloud) for [Kodi web interfaces/TeX](https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/).
> 
> Current translation status:

ahtuyi00-dotcom commented:
don't shone me

ahtuyi00-dotcom commented:
what's do me shone

weblate committed:
Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 100.0% (172 of 172 strings)

Deleted translation using Weblate (Ossetian (os_os))

Deleted translation using Weblate (Occitan (France) (oc_fr))

Translated using Weblate (Swedish (sv_se))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Croatian (hr_hr))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Bosnian (Bosnia and Herzegovina) (bs_ba))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 59.3% (102 of 172 strings)

Translated using Weblate (Ukrainian (uk_ua))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Ukrainian (uk_ua))

Currently translated at 11.0% (19 of 172 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 58.1% (100 of 172 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Swedish (sv_se))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Afrikaans (South Africa) (af_za))

Currently translated at 1.1% (2 of 172 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 23.8% (41 of 172 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Asturian (Spain) (ast_es))

Currently translated at 26.7% (46 of 172 strings)

Translated using Weblate (Slovenian (sl_si))

Currently translated at 1.1% (2 of 172 strings)

Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 51.1% (88 of 172 strings)

Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 51.1% (88 of 172 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 97.6% (168 of 172 strings)

Translated using Weblate (Dutch (nl_nl))

Currently translated at 31.9% (55 of 172 strings)

Translated using Weblate (Catalan (Spain) (ca_es))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Estonian (et_ee))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Estonian (et_ee))

Currently translated at 23.8% (41 of 172 strings)

Translated using Weblate (Catalan (Spain) (ca_es))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Japanese (ja_jp))

Currently translated at 24.4% (42 of 172 strings)

Translated using Weblate (Korean (ko_kr))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (German (de_de))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Italian (it_it))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (168 of 168 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (168 of 168 strings)

Co-authored-by: Alexey <signfinder@gmail.com>
Co-authored-by: Alfonso Cachero <alfonso.cachero@gmail.com>
Co-authored-by: Christian Gade <gade@kodi.tv>
Co-authored-by: Daniel Nylander <daniel@danielnylander.se>
Co-authored-by: Enol P <enolp@softastur.org>
Co-authored-by: FB <zhurong@qq.com>
Co-authored-by: Havok Dan <havokdan@yahoo.com.br>
Co-authored-by: Heiko Berner <berner.h@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: José Antonio Alvarado <jalvarado0.eses@gmail.com>
Co-authored-by: Kai Sommerfeld <ksooo@users.noreply.kodi.weblate.cloud>
Co-authored-by: Marek Adamski <fevbew@wp.pl>
Co-authored-by: Massimo Pissarello <mapi68@gmail.com>
Co-authored-by: Minho Park <parkmino@gmail.com>
Co-authored-by: Miniontoby <tobias.gaarenstroom@gmail.com>
Co-authored-by: Oskari Lavinto <olavinto@protonmail.com>
Co-authored-by: Pavlo Marianov <acid@jack.kyiv.ua>
Co-authored-by: Pere O <gotrunks@gmail.com>
Co-authored-by: SecularSteve <fairfull.playing@gmail.com>
Co-authored-by: Simon <dolenec@gmail.com>
Co-authored-by: Xavi Carol <xcarolr@gmail.com>
Co-authored-by: rimasx <riks_12@hot.ee>
Co-authored-by: roliverosc <roliverosc@hotmail.com>
Co-authored-by: skypichat <skypichat@hotmail.fr>
Co-authored-by: wabisabi926 <liwenliang926@163.com>
Co-authored-by: 复予 <clonewith@qq.com>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/af_za/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/ast_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/bs_ba/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/ca_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/de_de/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/es_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/et_ee/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/fi_fi/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/fr_fr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/hr_hr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/it_it/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/ja_jp/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/ko_kr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/nl_nl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/pl_pl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/pt_br/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/ru_ru/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/sl_si/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/sv_se/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/uk_ua/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/zh_Hans/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/zh_cn/
Translation: Kodi web interfaces/TeX

weblate committed:
Translated using Weblate (Czech (cs_cz))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Czech (cs_cz))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Czech (cs_cz))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 100.0% (172 of 172 strings)

Deleted translation using Weblate (Ossetian (os_os))

Deleted translation using Weblate (Occitan (France) (oc_fr))

Translated using Weblate (Swedish (sv_se))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Croatian (hr_hr))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Bosnian (Bosnia and Herzegovina) (bs_ba))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 59.3% (102 of 172 strings)

Translated using Weblate (Ukrainian (uk_ua))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Ukrainian (uk_ua))

Currently translated at 11.0% (19 of 172 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 58.1% (100 of 172 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Swedish (sv_se))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Afrikaans (South Africa) (af_za))

Currently translated at 1.1% (2 of 172 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 23.8% (41 of 172 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Asturian (Spain) (ast_es))

Currently translated at 26.7% (46 of 172 strings)

Translated using Weblate (Slovenian (sl_si))

Currently translated at 1.1% (2 of 172 strings)

Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 51.1% (88 of 172 strings)

Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 51.1% (88 of 172 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 97.6% (168 of 172 strings)

Translated using Weblate (Dutch (nl_nl))

Currently translated at 31.9% (55 of 172 strings)

Translated using Weblate (Catalan (Spain) (ca_es))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Estonian (et_ee))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Estonian (et_ee))

Currently translated at 23.8% (41 of 172 strings)

Translated using Weblate (Catalan (Spain) (ca_es))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Japanese (ja_jp))

Currently translated at 24.4% (42 of 172 strings)

Translated using Weblate (Korean (ko_kr))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (German (de_de))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Italian (it_it))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (172 of 172 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (168 of 168 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (168 of 168 strings)

Co-authored-by: Alexey <signfinder@gmail.com>
Co-authored-by: Alfonso Cachero <alfonso.cachero@gmail.com>
Co-authored-by: Christian Gade <gade@kodi.tv>
Co-authored-by: Daniel Nylander <daniel@danielnylander.se>
Co-authored-by: Enol P <enolp@softastur.org>
Co-authored-by: FB <zhurong@qq.com>
Co-authored-by: Havok Dan <havokdan@yahoo.com.br>
Co-authored-by: Heiko Berner <berner.h@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: José Antonio Alvarado <jalvarado0.eses@gmail.com>
Co-authored-by: Kai Sommerfeld <ksooo@users.noreply.kodi.weblate.cloud>
Co-authored-by: Marek Adamski <fevbew@wp.pl>
Co-authored-by: Massimo Pissarello <mapi68@gmail.com>
Co-authored-by: Michal Várady <miko.vaji@gmail.com>
Co-authored-by: Minho Park <parkmino@gmail.com>
Co-authored-by: Miniontoby <tobias.gaarenstroom@gmail.com>
Co-authored-by: Oskari Lavinto <olavinto@protonmail.com>
Co-authored-by: Pavlo Marianov <acid@jack.kyiv.ua>
Co-authored-by: Pere O <gotrunks@gmail.com>
Co-authored-by: SecularSteve <fairfull.playing@gmail.com>
Co-authored-by: Simon <dolenec@gmail.com>
Co-authored-by: Xavi Carol <xcarolr@gmail.com>
Co-authored-by: rimasx <riks_12@hot.ee>
Co-authored-by: roliverosc <roliverosc@hotmail.com>
Co-authored-by: skypichat <skypichat@hotmail.fr>
Co-authored-by: wabisabi926 <liwenliang926@163.com>
Co-authored-by: wabisabi926 <wabisabi926@foxmail.com>
Co-authored-by: 复予 <clonewith@qq.com>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/af_za/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/ast_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/bs_ba/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/ca_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/cs_cz/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/de_de/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/es_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/et_ee/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/fi_fi/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/fr_fr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/hr_hr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/it_it/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/ja_jp/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/ko_kr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/nl_nl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/pl_pl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/pt_br/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/ru_ru/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/sl_si/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/sv_se/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/uk_ua/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/zh_Hans/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-web-interfaces/tex/zh_cn/
Translation: Kodi web interfaces/TeX
